dali/test/python/test_operator_slice.py

```diff
@@ -678,3 +675,29 @@ def get_data():
 def test_slice_named_args_errors():
     yield check_slice_named_args_errors, 'cpu', 1
     yield check_slice_named_args_errors, 'gpu', 1
+
+def check_plain_memcpy(device, dtype, batch_size, num_threads):
```

Comment:
I find this test redundant. Isn't this case already covered? It's not like we can check that we're actually running plain memcpy here - and the separate test suggests just that. My recommendation is to remove it if existing tests cover that code path. Also, please verify manually (with debugger or a debug print) that the memcpy code path is actually selected.